libraries/ae-security/src/main/java/org/metaeffekt/core/security/cvss/CvssVector.java

```diff
@@ -403,7 +403,9 @@ public static CvssVector parseVector(String vector) {
                 return potentialCvss4P0Vector;
             }
 
-            LOG.warn("Cannot fully determine CVSS version in vector [{}]", vector);
+            if (vector.startsWith("CVSS:")) {
```

Comment:
The advantage of this approach is that it generally works for anything non-CVSS. Note that it's safe to check for the prefix here although CVSSv2 does not define any prefix because CVSSv2 has been handled before.

Reply:
This version would work for certain, and I see no drawbacks from this as you already mentioned.